scanner/src/main/kotlin/scanners/scanoss/ScanOss.kt

```diff
@@ -47,30 +56,79 @@ class ScanOss internal constructor(
             ScanOss(scannerName, scannerConfig, downloaderConfig)
     }
 
+    private val config = ScanOssConfig.create(scannerConfig).also {
+        log.info { "The $scannerName API URL is ${it.apiUrl}." }
+    }
+
+    private val service = ScanOssService.create(config.apiUrl)
+
+    companion object {
+        // The name of the multipart body being sent. Has no importance.
+        private const val FAKE_WFP_FILE_NAME = "anon.wfp"
```

Comment:
For consistency with the variable name, how about using "fake.wfp" as the file name?

Reply:
Also, at some point we IIRC decided with @fviernau that only public constants should become part of companion objects for discoverability. Private constants should become top-level in the file.